TypeCobol.Test/Parser/Preprocessor/ReplaceTestFiles/PgmReplacePartialTC.cbl

```diff
@@ -0,0 +1,25 @@
+﻿000001 01 :TAG:.
+000002   02 :TAG:-TOTO.
+000003   02 TOTO-:TAG:.
+000004   02 TO-:TAG:-TO.
+000005 REPLACE == :TAG: == BY == TITI ==.
+000001 01 :TAG:.
+000002   02 :TAG:-TOTO.
+000003   02 TOTO-:TAG:.
+000004   02 TO-:TAG:-TO.
+000005 REPLACE == :TAG: == BY == TITI ==
+000005         == 02 == BY == 05 ==
+000005         == :PREFIX: == BY == TEST ==.
+000001 01 :TAG:.
+000002   02 :PREFIX:-TOTO.
+000003   02 TOTO-:TAG:.
+000004   02 TO-:PREFIX:-TO.
+000001 REPLACE ==:tag:== BY ==var==.
+000002 01 :tag::tag: PIC X.
+000003 01 S:tag::tag: PIC X.
+000004 01 :tag:M:tag: PIC X.
+000005 01 :tag::tag:E PIC X.
+000006 01 S:tag:M:tag: PIC X.
+000007 01 :tag:M:tag:E PIC X.
+000008 01 S:tag::tag:E PIC X.
+000009 01 S:tag:M:tag:E PIC X.
```

Comment:
Content is duplicated from PgmReplacePartial.cbl, is it possible to keep a single source file and make a distinction only for result file ?

Reply:
It Seems that no :-)

Reply:
Ok I've found what's going on it's because of the cache of imported documents. I've tried this and it works :
- delete PgmReplacePartialTC.cbl
- alter PgmReplacePartialTC.txt first line with "Document path PgmReplacePartial"
- use this code in `CheckReplacePartial`

```csharp
            string testName = "PgmReplacePartial";
            PreprocessorUtils.ReplaceProject.CompilationOptions.IsCobolLanguage = cobol;
            PreprocessorUtils.ReplaceProject.ClearImportedCompilationDocumentsCache();
            string result = PreprocessorUtils.ProcessReplaceDirectives(testName);
            PreprocessorUtils.ReplaceProject.CompilationOptions.IsCobolLanguage = false;
            PreprocessorUtils.CheckWithReplaceResultFile(result, testName + (cobol ? string.Empty : "TC"));
```